english/blackout/blackout.md

```diff
@@ -1 +1,2 @@
-You walk down the tunnel and feel faint. You blackout and right before you hit the floor someone catches you.
\ No newline at end of file
+You walk down the tunnel and feel faint. You blackout and right before you hit the floor someone catches you.
+While you were blacked out you recall something you were [hypnotized to forget.](hypnotized-to-forget\hypnotized.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.